### PR TITLE
WIPP Sample Integration

### PR DESCRIPTION
**Pull request by mmwilder26**
Waste Isolation Pilot Plan (WIPP) Sample Integration

### CODE REVIEW

**Comment by mmwilder26** [comment]
Hey @JAG-UK  receiving message file reformatting under integrity checks . . . 

The above says 2 files reformatted then states 2 files must be reformatted. . . not sure which two files or why this message is being received.  Wondering if you have additional insight.

Run export PYTHONPATH=samples:${PYTHONPATH}
  export PYTHONPATH=samples:${PYTHONPATH}
  pycodestyle --format=pylint archivist_samples
  python3 -m pylint --rcfile=pylintrc archivist_samples
  black archivist_samples
  modified=$(git status -s | wc -l)
  if [ $modified -gt 0 ]
  then
      echo "there are $modified files that must be reformatted"
      exit 1
  fi
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.9.6/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.9.6/x64/lib

------------------------------------
Your code has been rated at 10.00/10

reformatted archivist_samples/wipp/main.py
reformatted archivist_samples/wipp/run.py
All done! ✨ 🍰 ✨

2 files reformatted, 51 files left unchanged.
there are 2 files that must be reformatted
Error: Process completed with exit code 1.

**Comment by leflambeur** [comment]
I took this locally and tried running format, got the same issue, I ran it with `-v` to get more details and it then simply worked, checked the diff and only very limited changes occured with little to no integrity issues - I have submitted the changes

Everything seems okay but please do verify as well

**Comment by mmwilder26** [comment]
Thanks for the info. . . I make the format changes locally to test on my end and go from there.

File: archivist_samples/wipp/run.py [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+        },
+        custom_attrs={
+            "wipp_sensors_shock": "0",
+            "wipp_sensors_rad": "45",
+        },
+        attachments=[
+            upload_attachment(arch, "truck_departure.jpg", "arc_primary_image")
+        ],
+    )
+    LOGGER.info("Waypoint registered...")
```
Comment (JAG-UK) [inline archivist_samples/wipp/run.py:223]: This needs to actually register waypoint data. That means:
* The event needs to actually have a GIS location on it
* There need to be lots of them! The route is specified in the attachment to the first transit run

So you need a loop that records these waypoints:

    waypoints = [["Atlanta", "33.592177", "-84.406064"],
                 ["Talladega", "33.592177", "-86.248379"],
                 ["Birmingham", "33.494993", "-86.895403"],
                 ["Tuscaloosa", "33.184220", "-87.610330"],
                 ["Meridian", "32.391672", "-88.532850"],
                 ["Jackson", "32.285409", "-90.074633"],
                 ["Monroe", "32.463868", "-91.893769"],
                 ["Shreveport", "32.537993", "-93.651582"],
                 ["Tyler", "32.334001", "-95.321504"],
                 ["South Dallas", "32.639816", "-96.826631"],
                 ["Gordon", "32.499115", "-98.521317"],
                 ["Abilene", "32.457004", "-99.816598"],
                 ["Big Spring", "32.244259", "-101.458984"],
                 ["Andrews", "32.312469", "-102.548197"],
                 ["Seminole", "32.457004", "-99.816598"],
                 ["Hobbs", "32.244259", "-101.458984"]]

**Review by JAG-UK — CHANGES_REQUESTED** [review]

**Review by mmwilder26 — COMMENTED** [review]
Hey @JAG-UK  Added location for waypoint.  Also added latitude and longitude but not showing as event attributes.  Double checked w/in the UI logging in with Jill and did not see latitude or longitude.  Is this info required?

File: archivist_samples/wipp/run.py (hunk 1 above)
Comment (JAG-UK) [reply archivist_samples/wipp/run.py]: Checked a new run, looks good!

**Review by JAG-UK — APPROVED** [review]

**Pull request merged**